client/src/theme.js

```diff
@@ -65,6 +65,12 @@ const theme = createTheme({
     subtitle: { fontSize: "1.125rem" },
     body1: { fontSize: "1.125rem" },
     body2: { fontSize: "1.125rem", color: "#C4C4C4" },
+    navText: {
+      fontSize: ".8rem",
+      fontWeight: 600,
+      color: "#2F3032",
```

Comment:
is this a theme color we can reference rather than hard code?

Reply:
resolving since comment belows is similar